data/json/items/ammo/nail.json

```diff
@@ -15,7 +15,7 @@
     "type": "AMMO",
     "category": "spare_parts",
     "name": "nail",
-    "description": "A box of nails, mainly useful with a hammer.",
+    "description": "An pin-shaped piece of iron, can be used to construct various things with a hammer.",
```

Comment:
I'm not sure that ```An``` is a correct form of indefinite article in this case.

Reply:
It probably is incorrect. The trouble with English as a second language I guess. Will fix it.